### ISSUE TITLE
The Pull requests review time is too long

### ISSUE THREAD

**Issue by hasanbayatme** [vlucas/devdata.io#24]
I have submitted two pull requests but they won't accepted for a long time.

Please take a look at [Pull requests](https://github.com/vlucas/devdata.io/pulls) and accept them.

Thanks.

**Comment by vlucas** [comment]
Yes, I apologize. I am going through them now and updating the site.
